packages/components/scripts/findImportedTypes.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-console */
```

Comment:
[Suggestion] Add this script in the package.json and explain why one might use it in the readme?